ev3dev/core.py

```diff
@@ -955,6 +958,94 @@ def wait_while(self, s, timeout=None):
         """
         return self.wait(lambda state: s not in state, timeout)
 
+    def set_power(self, power):
+        self.speed_sp = power
+
+    def set_position_rotations(self, power, rotations):
```

Comment:
What are these methods (setting the `position_sp`, `time_sp`, etc.) for? They don't seem to match up with the EV3-G blocks in the spec image.

Reply:
Originally in `on_for_degrees`, etc I just had a `run_to_abs_pos()` call where I passed in all of needed kwargs.  This worked fine for a single motor but for MoveTank and MoveSteering if I called
```
    self.left_motor.on_for_degrees(...)
    self.right_motor.on_for_degrees(...)
```
it added delay between when the two motors start.  We want to follow the model of "set as many parameters as you can for both motors, then start the motors" so that they start as close together as possible.  So that is why set_power(), etc are broken out like this, it is so I can set all of the parameters first when working with pairs of motors.

Reply:
How about making them private (i.e. leading underscore)?

Reply:
yep can make them private